src/app/GitUI/Editor/Diff/Segment.cs

```diff
@@ -0,0 +1,9 @@
+﻿using ICSharpCode.TextEditor.Document;
+
+namespace GitUI.Editor.Diff;
+
+public struct Segment : ISegment
```

Comment:
Why do we need the interface?

Reply:
It is a TextEditor interface

Reply:
There is one implementation in externals\ICSharpCode.TextEditor\Project\Src\Document\AbstractSegment.cs
Dis not use it due to the name.
The class is only used in this file.